app/kudos/templates/share.html

```diff
@@ -0,0 +1,142 @@
+{% load i18n static %}
+{% load kudos_extras %}
+<style>
+  .modal {
+    display: none;
+    position: fixed;
+    z-index: 1025;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(0,0,0);
+    background-color: rgba(0,0,0,0.4);
+    max-width: 100%;
+    overflow: scroll;
+  }
+
+  .modal-content {
+    background-color: #fefefe;
+    margin: 15% auto;
+    padding: 20px;
+    border: none;
+    border-radius: 0;
+    width: 60%;
+  }
+
+  .fa-times:hover,
+  .fa-times:focus {
+    cursor: pointer;
+  }
+
+  .modal #kudos-image {
+    max-width: 12rem;
+  }
+
+  .tooltip-share {
+    z-index: 1036;
+  }
+
+  #share-modal button i {
+    font-style: normal;
+    color: #fff;
+  }
+
+  #shareLink {
+    background-color: #fff;
+    border: 1px solid #979797;
+    min-width: 18rem;
+    width: auto;
+  }
+
+  .tooltip-share .tooltip-inner {
+    padding: 5px;
+    width: 150px;
+  }
+
+  .tooltip-share .kudos_msg {
```

Comment:
I think this is not used, since this class was created by Kevin to do some bobble messages from kudos (like kudos talking) , and didn't see any html elem with this in the share html.